src/gd_rotate.c

```diff
@@ -362,3 +362,210 @@ gdImagePtr gdImageRotate270 (gdImagePtr src, int ignoretransparent)
 
 	return dst;
 }
+
+/* Rotates an horizontal flipped image by 90 degrees (counter clockwise) */
+gdImagePtr gdImageFlipHRotate90 (gdImagePtr src, int ignoretransparent)
+{
+	int uY, uX;
+	int c,r,g,b,a;
+	gdImagePtr dst;
+	FuncPtr f;
+
+	if (src->trueColor) {
+		f = gdImageGetTrueColorPixel;
+	} else {
+		f = gdImageGetPixel;
+	}
+	dst = gdImageCreateTrueColor(src->sy, src->sx);
+	if (dst != NULL) {
+		int old_blendmode = dst->alphaBlendingFlag;
+		dst->alphaBlendingFlag = 0;
+
+		dst->transparent = src->transparent;
+
+		gdImagePaletteCopy (dst, src);
```

Comment:
I'm curious that I need to apply these changes because code mentioned in below is copied from gdImageRotate90 which was already exist. 

What I change were new image's resolution and dest pixel position.

Reply:
you're adding new code.  new code shouldn't be wrong.  the fact that some existing code is wrong isn't a valid reason to add more wrong code.